nwb_web_gui/dashapps/pages/converter/converter_utils/forms.py

```diff
@@ -301,51 +300,44 @@ def get_definitions_items(self, ref_key, extra_key, sublist=False):
 
         return form
 
-    def get_string_field_input(self, value, input_id):
+    @staticmethod
```

Comment:
The `staticmethod` decorator should be used for all methods that do not access `self`.